dev-lang/vala/vala-0.36.11.recipe

```diff
@@ -64,6 +73,20 @@ BUILD_PREREQUIRES="
 	cmd:pkg_config$secondaryArchSuffix
 	"
 
+defineDebugInfoPackage vala$secondaryArchSuffix \
+	$libDir/libvala-${portVersion%.*}.so.0.0.0 \
```

Comment:
Shall we use a `libVersion` (and `libVersionCompat`)?
```diff
-	$libDir/libvala-${portVersion%.*}.so.0.0.0
+	$libDir/libvala-${portVersion%.*}.so.$libVersion
```
(Of course, if you define `libVersion` you will want to replace all other occurences of `0.0.0 compat >= 0` by `$libVersionCompat` and `0.0.0` by `$libVersion`.)

Reply:
Also leftover from last version, haven't looked into libVersion here yet :)